src/popup/components/buttons/BtnMain.vue

```diff
@@ -130,6 +130,8 @@ export default defineComponent({
 
   &.secondary {
     --color: #{variables.$color-grey-medium};
+
+    padding: 0 32px;
```

Comment:
I suggest to use CSS variable fox X axis paddings to avoid changing the Y axis paddings to 0.